torch/csrc/jit/codegen/cuda/arith.cpp

```diff
@@ -731,6 +745,56 @@ TensorView* clamp(TensorView* in, Val* min_val, Val* max_val) {
   return clamp(in->as<Val>(), min_val, max_val)->as<TensorView>();
 }
 
+// sum_to operator
+
+// Name of sum_to is different from NV fuser naming,
+// this is to align with the operator name
+TensorView* sum_to(TensorView* v1, const std::vector<Int*>& shape) {
+  const auto& v1_root = TensorDomain::noReductions(v1->getRootDomain());
+
+  TORCH_CHECK(
+      v1_root.size() >= shape.size(),
+      "sum_to: Error trying to reduce",
+      v1,
+      "into a shape of size",
+      shape.size());
+
+  // If no reduction is needed sum_to returns the input tv
+  TensorView* v2 = v1;
```

Comment:
can we use more descriptive names than `v1` and `v2`?

Reply:
Thanks for the suggestion. Renamed them to `in` and `out`.